src/dmd/utf.d

```diff
@@ -19,15 +19,15 @@ nothrow pure @nogc:
 bool utf_isValidDchar(dchar c)
 {
     // TODO: Whether non-char code points should be rejected is pending review
-    // largest character code point
-    if (c > 0x10FFFF)
-        return false;
+
```

Comment:
Nit: two lines of whitespace.

Reply:
Updated to include an extra line of whitepace.

Reply:
Ha, I actually meant the other one other one below the second comment was sufficient and that one was unnecessary.